controller/api/destination/server.go

```diff
@@ -7,23 +7,28 @@ import (
 	"strings"
 
 	pb "github.com/linkerd/linkerd2-proxy-api/go/destination"
-	"github.com/linkerd/linkerd2/controller/api/util"
+	"github.com/linkerd/linkerd2/controller/api/destination/watcher"
 	discoveryPb "github.com/linkerd/linkerd2/controller/gen/controller/discovery"
 	"github.com/linkerd/linkerd2/controller/k8s"
-	"github.com/linkerd/linkerd2/pkg/addr"
 	"github.com/linkerd/linkerd2/pkg/prometheus"
-	log "github.com/sirupsen/logrus"
+	logging "github.com/sirupsen/logrus"
```

Comment:
we call this import `log` in every use. I don't think it should be different in this module. If you feel strongly about the `logging` change, I'd prefer we make that its own change on the whole project.

Reply:
I did this so that I can have variables named `log`.  What do you call `log` if you can't call it `log`?

Reply:
idiomatic go would probably call them `l` or something...  in general i've noticed this pattern _especially_ when the field/var name would just be the type/module name. But i'd prefer to do whatever we do elsewhere in the codebase.

Reply:
`logrus.Entry` is not used elsewhere in the project, except for one usage in `cni-plugin` where the logrus import is left as `logrus`